dashboard/lib/services/curriculum_pdfs.rb

```diff
@@ -62,6 +92,7 @@ def self.generate_pdfs?(script_data)
       !timestamps_equal(new_timestamp, existing_timestamp)
```

Comment:
ah, yeah, I should be more specific here. `serialized_at` is derived from `updated_at` and is used interchangeably at a couple of different points, just not at this one

Reply:
Interesting. I'm not sure where it belongs, but it would be great if you could document if/why serialized_at is needed. FWIW some changes went in recently that made it so a bit more stuff triggers touching script.updated_at: https://github.com/code-dot-org/code-dot-org/blob/fdb47d146e19b189399d9e8d82f5e3da970c60f1/dashboard/app/models/script.rb#L1076-L1082

Reply:
actually I think I'm going to have to backtrack on challenging whether serialized_at is needed -- saving a lesson edit page, for example, would regenerate script json but wouldn't touch script.updated_at. so in that case, I'm interested in more information regarding your comment about when serialized_at and updated_at might be used interchangeably, since it sounds like they do represent different things.

Reply:
Hah, this is why it's a good idea to write documentation at the same time you write the code, and not months later. You're right, `serialized_at` is in fact not derived from `updated_at`; that's just an approach I considered at one point and forgot that I didn't take. It does fulfill a very similar role, which is why I (incorrectly) occasionally use them interchangeably